jenkins_build.sh

```diff
@@ -0,0 +1,47 @@
+set -e
```

Comment:
Please add `#!/bin/bash` to the first line just to avoid any potential confusion.

Also, add `-x` so in case something really weird goes wrong in the future, the job log in Jenkins will have a more complete record of what the script did.